pkg/elfparser/elf.go

```diff
@@ -408,6 +455,13 @@ func (e *elfLoader) parseMap() ([]ebpf_maps.CreateEBPFMapInput, error) {
 			}
 		}
 		log.Infof("Found map name %s", mapData.Name)
+
+		if len(customData.CustomMapSize) != 0 {
```

Comment:
I think, it might be safe to validate the size value passed in. Cap it @ 1M with maybe the lower bound being @ 64k

Reply:
Oh this is just checking if custom Map is present

Reply:
I mean, once we identify we've a non Zero value in `CustomMapSize`, I'm proposing we cap min and max values before we set the `mapData.MaxDntries` field below. We can probably cap the configurable value in the client as well (i.e.,) NP agent ..

Reply:
Yeah I think the caller can check since this is SDK